src/utils/getUserById.ts

```diff
@@ -0,0 +1,5 @@
+import usersFromServer from '../api/users';
+
+export const getUserById = (id: number) => {
+  return usersFromServer.find(user => user.id === id);
```

Comment:
Consider handling the case where no user is found with the specified ID. You could return `null` or a default user object to prevent potential issues when the function returns `undefined`.